### PR TITLE
[Vectorize] add more pricing examples

### PR DESCRIPTION
**Pull request by elithrar**
This PR:

- Adds pricing examples to Vectorize.
- Adds a link to the LangChain module (future: fuller example within our docs) - https://js.langchain.com/docs/modules/data_connection/vectorstores/integrations/cloudflare_vectorize

### CODE REVIEW

File: content/vectorize/platform/pricing.md [hunk 1]
```diff
@@ -23,6 +23,22 @@ You are not billed for CPU, memory, "active index hours", or the number of index
 
 {{<render file="_vectorize-pricing.md" productFolder="workers">}}
 
+### Usage examples
+
+The following table defines a number of example use-cases and the estimated monthly cost for querying a Vectorize index. These estimates do not include the Vectorize usage that's part of the
```
Comment (Maddy-Cloudflare) [inline content/vectorize/platform/pricing.md]:
```suggestion
The following table defines a number of example use-cases and the estimated monthly cost for querying a Vectorize index. These estimates do not include the Vectorize usage that is part of the 
```

Comment (Maddy-Cloudflare) [reply content/vectorize/platform/pricing.md]: @elithrar this sentence is incomplete.

File: content/vectorize/platform/pricing.md [hunk 2]
```diff
@@ -23,6 +23,22 @@ You are not billed for CPU, memory, "active index hours", or the number of index
...
+| ---------------------- | ---------------------- | -------------------- | -------------------- | ------------------- | -------------------- |
+| Experiment             | 384                    | 5,000 vectors        | 10,000               | `(5000+10000)*384*(0.040/1000000)` | $0.24 / mo <sup>included</sup> |
+| Scaling                | 768                    | 25,000 vectors       | 50,000               | `(25000+50000)*768*(0.040/1000000)` | $2.31 / mo <sup>partial</sup> |
+| Production             | 768                    | 50,000 vectors       | 200,000               | `(50000+200000)*768*(0.040/1000000)` | $7.68 / mo |
+| Large                  | 768                    | 250,000 vectors       | 500,000               | `(250000+500000)*768*(0.040/1000000)` | $23.04 / mo |
+| XL                     | 1536                   | 500,000 vectors      | 1,000,000             | `(500000+1000000)*1536*(0.040/1000000)` | $92.16 / mo |
+
+<sup>included</sup> All of this usage would fall into the Vectorize usage included within the Workers free or paid plan.
```
Comment (Maddy-Cloudflare) [inline content/vectorize/platform/pricing.md]:
```suggestion
<sup>included</sup> All of this usage would fall into the Vectorize usage included in the Workers Free or Paid plan.
```

File: content/vectorize/platform/pricing.md [hunk 3]
```diff
@@ -23,6 +23,22 @@ You are not billed for CPU, memory, "active index hours", or the number of index
...
+
+<sup>most</sup> Most of this usage would fall into the Vectorize usage included within the Workers paid plan.
```
Comment (Maddy-Cloudflare) [inline content/vectorize/platform/pricing.md]:
```suggestion
<sup>most</sup> Most of this usage would fall into the Vectorize usage included within the Workers Paid plan.
```

**Review by Maddy-Cloudflare — APPROVED** [review]
Line 28: that sentence is incomplete.

File: content/vectorize/platform/pricing.md (hunk 1 above)
Comment (elithrar) [reply content/vectorize/platform/pricing.md]: Thanks - fixed!

File: content/vectorize/platform/pricing.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ You are not billed for CPU, memory, "active index hours", or the number of index
 
 ### Usage examples
 
-The following table defines a number of example use-cases and the estimated monthly cost for querying a Vectorize index. These estimates do not include the Vectorize usage that's part of the 
+The following table defines a number of example use-cases and the estimated monthly cost for querying a Vectorize index. These estimates do not include the Vectorize usage that's part of the Workers free and paid plans.
```
Comment (Maddy-Cloudflare) [inline content/vectorize/platform/pricing.md]:
```suggestion
The following table defines a number of example use-cases and the estimated monthly cost for querying a Vectorize index. These estimates do not include the Vectorize usage that is part of the Workers Free and Paid plans.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/vectorize/platform/pricing.md [hunk 5]
```diff
@@ -23,6 +23,22 @@ You are not billed for CPU, memory, "active index hours", or the number of index
...
+| ---------------------- | ---------------------- | -------------------- | -------------------- | ------------------- | -------------------- |
+| Experiment             | 384                    | 5,000 vectors        | 10,000               | `(5000+10000)*384*(0.040/1000000)` | $0.24 / mo <sup>included</sup> |
+| Scaling                | 768                    | 25,000 vectors       | 50,000               | `(25000+50000)*768*(0.040/1000000)` | $2.31 / mo <sup>partial</sup> |
+| Production             | 768                    | 50,000 vectors       | 200,000               | `(50000+200000)*768*(0.040/1000000)` | $7.68 / mo |
+| Large                  | 768                    | 250,000 vectors       | 500,000               | `(250000+500000)*768*(0.040/1000000)` | $23.04 / mo |
+| XL                     | 1536                   | 500,000 vectors      | 1,000,000             | `(500000+1000000)*1536*(0.040/1000000)` | $92.16 / mo |
+
+<sup>included</sup> All of this usage would fall into the Vectorize usage included in the Workers Free or Paid plan.
+
+<sup>most</sup> Most of this usage would fall into the Vectorize usage included within the Workers Paid plan.
```
Comment (irvinebroque) [inline content/vectorize/platform/pricing.md:40]: This is clever, we should do it with all the usage-based products where some amount of usage is included in the paid plan.

@tanushree-sharma re: https://blog.cloudflare.com/running-serverless-puppeteer-workers-durable-objects/

**Review by irvinebroque — APPROVED** [review]

**Pull request merged**